app/dashboard/views.py

```
@@ -1049,7 +1045,7 @@ def profile_keywords(request, handle):
     """
     try:
         profile = profile_helper(handle, True)
-    except ProfileNotFoundException:
+    except (ProfileNotFoundException, ProfileHiddenException):
```

Catch all exceptions raised by `profile_helper`